tools/tune/eq/eq_get_abi.m

```diff
@@ -11,18 +11,55 @@
 %
 % Author: Seppo Ingalsuo <seppo.ingalsuo@linux.intel.com>
 
-%% Use sof-ctl to write ABI header into a file
+ipc4_use_sofctl = 0; % Set to 1 when sof-ctl update is merged
 abifn = 'eq_get_abi.bin';
-cmd = sprintf('sof-ctl -g %d -b -o %s > /dev/null', setsize, abifn);
-system(cmd);
+switch ipc_ver
+	case 3,
+		%% Use sof-ctl to write ABI header into a file
+		cmd = sprintf('sof-ctl -g %d -b -o %s', setsize, abifn);
+		[bytes, nbytes] = get_abi_with_sofctl(cmd, abifn);
+	case 4,
+		if ipc4_use_sofctl
+			cmd = sprintf('sof-ctl -I IPC4 -g %d -b -o %s', setsize, abifn);
```

Comment:
Probably this statement can't reach

Reply:
I'll change ipc4_use_sofctl to 1 when Libin's patch is merged. It's there as reminder to do it and remove the hard-coded header from Matlab code.